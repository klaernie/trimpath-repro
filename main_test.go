package main

import (
	"testing"

	"github.com/gkampitakis/go-snaps/snaps"
)

func TestRun(t *testing.T) {
	snaps.MatchSnapshot(t, run())
}
