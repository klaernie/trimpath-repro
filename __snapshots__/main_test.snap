
[TestRun - 1]
Hello World
---
